src/lib.rs

```diff
@@ -212,6 +212,8 @@ pub trait BitSetLike {
         }
     }
 
+    fn remove(&mut self, i: Index) -> bool;
```

Comment:
this is truly unsound - this trait API is otherwise immutable. Maybe move it to a `DrainableBitSet` trait?

Reply:
It's not unsound per say - you just cannot implement it for shared references.
That's actually good idea. I'll create trait like that.